src/metrics/tests/test_scopus_citations.py

```diff
@@ -25,6 +26,27 @@ def test_scopus_parse_entry(self):
         expected_entries = per_page * len(response_fixtures)
         self.assertEqual(expected_entries, len(res))
 
+    def test_scopus_parse_bad_entry(self):
+        "entries with bad values are written to disk to be inspected later"
+        with patch('metrics.handler.writefile') as mock:
+            entry = self.good_entry
+            # skitch entry so it's bad
+            entry['prism:doi'] = 'PAAAAAANTS'
+            result = citations.parse_entry(entry)
+            self.assertTrue('bad' in result) # ll: {'bad': ...}
+            self.assertTrue(mock.called)
+
+    def test_scopus_unparseable_entry(self):
+        "unparseable entries have a different return type"
+        with patch('metrics.handler.writefile') as mock:
+            entry = self.good_entry
+            # skitch entry so it's bad
+            del entry['prism:doi']
+            result = citations.parse_entry(entry)
+            self.assertTrue('bad' in result) # ll: {'bad': ...}
+            # a bad entry is still returned but the body isn't written to disk for inspection
```

Comment:
I wonder why we don't store anything for this case?

Reply:
if the record is missing fields, it's hard to create the minimum viable Article object. Scopus is returning about 23 of these I think, so 23 errors every day we can't recover from. In this case, I'm plain ignoring them. Well, a warning is issued, but that's about it. 

If those fields are present though and have bad values, there is room for wrangling, like the '10.7554/elife.e123456' formatted doi I came across the other day.